Framework/MDAlgorithms/src/ConvertToDiffractionMDWorkspace2.cpp

```diff
@@ -100,12 +104,13 @@ void ConvertToDiffractionMDWorkspace2::init() {
       "order to merge them later\n");
   setPropertyGroup("MinRecursionDepth", getBoxSettingsGroupName());
 
-  std::vector<double> extents{-50, +50};
+  std::vector<double> extents;
   declareProperty(
```

Comment:
Do we want to create a new **version** of this algorithm for this change? This is going to give different results now.